nextpack/scripts/init.cjs

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
```

Comment:
Maybe rename this to `sync.cjs`, since it needs to be run every time the workspaces are modified.